site/components/ViewAllContents.tsx

```diff
@@ -977,7 +979,9 @@ const ViewAllContents = ({
                                 defaultDisplay="Select a mode"
                                 selectValues={[
                                     { display: "Any", value: "any" },
-                                    ...VEHICLE_MODES.sort((a, b) => a.display.localeCompare(b.display)),
+                                    ...VEHICLE_MODES.filter((v) =>
+                                        showUnderground ? true : v.value !== VehicleMode.underground,
```

Comment:
We're doing this same filter expression in a lot of places, if possible it might be nice to bring this out into a separate utils function and reference that function throughout the codebase. Maybe call it something like `filterVehicleModes`

Reply:
done